keyboards/redscarf_iiplus/verd/rules.mk

```diff
@@ -0,0 +1,35 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+BOOTLOADER = qmk-dfu
```

Comment:
This is probably actually `atmel-dfu`, though, I don't think it actually makes a huge difference.

Reply:
apparently uses lufa

Reply:
Are you able to reset the EEPROM from the QMK Toolbox? If not, it's atmel-dfu.

Reply:
can't really interact with it with QMK toolbox, have to load the hex file with some sort of web app. It's kinda weird